src/PersonObjects/Player/PlayerObjectGeneralMethods.tsx

```diff
@@ -933,6 +933,8 @@ export function startFactionSecurityWork(this: IPlayer, faction: Faction): void
 export function workForFaction(this: IPlayer, numCycles: number): boolean {
   const faction = Factions[this.currentWorkFactionName];
 
+  if (!faction) return false
```

Comment:
I'm a big fan of adding `{}` even for single line `if`s. And you missed the `;`

Reply:
yeah fair enough 👍 ill adjust later today.

opps must not have hit save for autolint to kick in